cmd/server/server.go

```diff
@@ -50,6 +50,7 @@ type serverArgs struct {
 	// mixer manager args
 	serviceConfigFile      string
 	globalConfigFile       string
+	kubeConfig             string
```

Comment:
I can help with this binary in Manager repo. It was our original design goal not to couple Mixer with platform specific mechanisms. You need to provide a way to reload configuration in Mixer. There are two generic methods to achieve this:
1) you expose an HTTP endpoint that accepts a config JSON as a POST body
2) you listen for SIG HUP in your process, atomically displace files, and trigger a reload.
Please implement one of these methods.